mmdet/datasets/openimages.py

```diff
@@ -33,7 +33,7 @@ def __init__(self,
         """
         Args:
             label_description_file (str): File path to the label map proto.
-            need_get_father (bool): Whether get father class of the current
+            get_parent_class (bool): Whether get father class of the current
```

Comment:
```suggestion
            get_parent_class (bool): Whether get parent class of the current
```
Remember to modify the docstring.